Documentation/config/core.txt

```diff
@@ -195,6 +195,7 @@ core.autocrlf::
 	working directory and the repository has LF line endings.
 	This variable can be set to 'input',
 	in which case no output conversion is performed.
+	The default value is false.
```

Comment:
That is not correct, in general. On Windows, it is `true`.

Reply:
> That is not correct, in general. On Windows, it is true.

I tried on windows, without autocrlf.
default looks false:
![Screenshot](https://user-images.githubusercontent.com/4222502/77825518-2d9e7000-714d-11ea-84ce-91d516fdbf4b.png)

Reply:
When you install Git for Windows and leave all the default values, `core.autocrlf` is `true`. Most users will keep the defaults.

Reply:
I understood.
Installer sets autocrlf=true.
So if I remove the autocrlf settings, what is the value of autocrlf?
I want to know the default value.